test/Microsoft.Azure.SignalR.IntegrationTests/Infrastructure/MockServiceConnection.cs

```diff
@@ -13,12 +14,15 @@ namespace Microsoft.Azure.SignalR.IntegrationTests.Infrastructure
     /// </summary>
     internal class MockServiceConnection : IServiceConnection
     {
+        static int s_num = 0;
```

Comment:
Good eye... The monstrosity I use this for probably deserves a separate PR. 
But since we're going to tackle AllConnectionsEstablishedTask, I am going to need it here. Stay tuned...